### ISSUE TITLE
fix broken installation

### ISSUE THREAD

**Issue by xiaoxiae** [xiaoxiae/Florodoro#31]
The installation with `pip install ...` seems to be broken, for some reason...

**Comment by xiaoxiae** [comment]
Seems to be working okay when testing on a new laptop, closing for now.

**Comment by mprobson** [comment]
I just tried to install (and run) this today using both `pip install florodoro` and `pip install -i https://test.pypi.org/simple/ florodoro` and both broke with the same error:

```
$ florodoro
Traceback (most recent call last):
  File "/usr/local/anaconda3/bin/florodoro", line 5, in <module>
    from florodoro.__init__ import run
  File "/usr/local/anaconda3/lib/python3.8/site-packages/florodoro/__init__.py", line 19, in <module>
    from florodoro.history import History
  File "/usr/local/anaconda3/lib/python3.8/site-packages/florodoro/history.py", line 7, in <module>
    from florodoro.plants import Plant
  File "/usr/local/anaconda3/lib/python3.8/site-packages/florodoro/plants.py", line 5, in <module>
    from typing import Annotated, NewType, Callable
ImportError: cannot import name 'Annotated' from 'typing' (/usr/local/anaconda3/lib/python3.8/typing.py)
```

It seems that the released version does not match the version in the repo which removed `Annotated` from `plants.py`. I tried cloning and setting up this repo locally by doing the following:

```
git clone ...
cd Florodoro
python -m venv venv
source venv/bin/activate
python setup.py build
python setup.py install
cd build/lib/florodoro
python __init__.py
```

But I wasn't able to get it working since it couldn't load/open `PyQt5.QtCore`. Happy to help debug this if there was some more info on how to get a development version running locally.

P.S. I had to rename LICENSE to LICENSE.txt to get the above to work and I had lots of deprecation warnings as well. Happy to open a separate issue.

**Comment by xiaoxiae** [comment]
Sorry for the difficulties. The app is a bit neglected, I haven't updated it in quite a while (despite frequently using it).

I'll try to fix the installation over the weekend (and possibly close some other issues in the process), thanks for the ping!

**Comment by xiaoxiae** [comment]
I made some changes that I published on testing PyPi instance (version `0.7.1`). It can be installed by running
```
pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple florodoro
```

Everything seems to work fine on my end, but I'll do some more tests on other machines and systems before merging to master and closing the issue.

**Comment by mprobson** [comment]
No worries, thanks for making such a cool app! Whatever you did seems to have fixed it, launches no problem for me now.
